tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/TracerFlareTests.cs

```diff
@@ -19,7 +19,6 @@
 namespace Datadog.Trace.ClrProfiler.IntegrationTests;
 
 [CollectionDefinition(nameof(DynamicConfigurationTests), DisableParallelization = true)]
```

Comment:
Did you mean to delete this line instead?

```suggestion
[Collection(nameof(DynamicConfigurationTests))]
```